### ISSUE TITLE
TypeError when applied to arguments object

### ISSUE THREAD

**Issue by davidchambers** [thejameskyle/pretty-format#3]
Expected behaviour:

```javascript
prettyFormat((function() { return arguments; }(1, 2, 3)))
// => '(function() { return arguments; }(1, 2, 3))'
```

Actual behaviour:

```javascript
prettyFormat((function() { return arguments; }(1, 2, 3)))
// TypeError: Cannot read property 'length' of undefined
```

**Comment by thejameskyle** [comment]
Fixed and released in v1.1.1

**Comment by davidchambers** [comment]
You don't intend to differentiate arguments objects from plain objects?

**Comment by thejameskyle** [comment]
I could do that. I'd probably just do:

```js
Arguments [
  1,
  2,
  3
]
```

**Comment by davidchambers** [comment]
That seems reasonable.

**Comment by thejameskyle** [comment]
Released in v1.2.0.
